From 96d9307979423bfe84bd35d82655654b72f1d8ad Mon Sep 17 00:00:00 2001
Message-ID: <96d9307979423bfe84bd35d82655654b72f1d8ad.1701297273.git.sharlatanus@gmail.com>
In-Reply-To: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
References: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sun, 29 Oct 2023 22:18:50 +0000
Subject: [PATCH v3 46/66] gnu: go-github-com-aws-aws-sdk-go: Move to (gnu
 packages golang-web).

* gnu/packages/golang.scm (go-github-com-aws-aws-sdk-go): Move from
here...
* gnu/packages/golang-web.scm: ...to here.

Change-Id: Ic639856afb340c2a7aea6dc917d2bcd32290d1b7
---
 gnu/packages/golang-web.scm | 28 ++++++++++++++++++++++++++++
 gnu/packages/golang.scm     | 25 -------------------------
 2 files changed, 28 insertions(+), 25 deletions(-)

diff --git a/gnu/packages/golang-web.scm b/gnu/packages/golang-web.scm
index 1542c409fe..7ecdcbd8b5 100644
--- a/gnu/packages/golang-web.scm
+++ b/gnu/packages/golang-web.scm
@@ -6,6 +6,7 @@
 ;;; Copyright © 2020 Oleg Pykhalov <go.wigust@gmail.com>
 ;;; Copyright © 2020 Ryan Prior <rprior@protonmail.com>
 ;;; Copyright © 2020-2022 Efraim Flashner <efraim@flashner.co.il>
+;;; Copyright © 2021 Collin J. Doering <collin@rekahsoft.ca>
 ;;; Copyright © 2021 Raghav Gururajan <rg@raghavgururajan.name>
 ;;; Copyright © 2021 Sarah Morgensen <iskarian@mgsn.dev>
 ;;; Copyright © 2022 Adam Kandur <kefironpremise@gmail.com>
@@ -104,6 +105,33 @@ (define-public go-github-com-aws-sdk
      "This is the official AWS SDK for the Go programming language.")
     (license license:asl2.0)))
 
+;; XXX: This package might be a duplicate of go-github-com-aws-sdk, it's not
+;; in use anywhere. Keep it here for the farther review.
+(define-public go-github-com-aws-aws-sdk-go
+  (package
+    (name "go-github-com-aws-aws-sdk-go")
+    (version "1.36.18")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/aws/aws-sdk-go")
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "169mkkw1cff1px6326krwvfpfj07sb4y5rbn003gi4bk176h6ry9"))))
+    (build-system go-build-system)
+    (propagated-inputs
+     (list go-github-com-jmespath-go-jmespath))
+    (arguments
+     '(#:import-path "github.com/aws/aws-sdk-go"
+       #:phases %standard-phases))
+    (synopsis "The official AWS SDK for the Go programming language")
+    (description
+     "The official AWS SDK for the Go programming language.")
+    (home-page "https://github.com/aws/aws-sdk-go")
+    (license license:asl2.0)))
+
 (define-public go-github-com-aws-aws-sdk-go-v2
   (package
     (name "go-github-com-aws-aws-sdk-go-v2")
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index a02da0fb49..d631973702 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -11550,31 +11550,6 @@ (define-public go-github-com-mtibben-androiddnsfix
 (define-public go-github-com-androiddnsfix
   (deprecated-package "go-github-com-androiddnsfix" go-github-com-mtibben-androiddnsfix))
 
-(define-public go-github-com-aws-aws-sdk-go
-  (package
-    (name "go-github-com-aws-aws-sdk-go")
-    (version "1.36.18")
-    (source (origin
-              (method git-fetch)
-              (uri (git-reference
-                    (url "https://github.com/aws/aws-sdk-go")
-                    (commit (string-append "v" version))))
-              (file-name (git-file-name name version))
-              (sha256
-               (base32
-                "169mkkw1cff1px6326krwvfpfj07sb4y5rbn003gi4bk176h6ry9"))))
-    (build-system go-build-system)
-    (propagated-inputs
-     (list go-github-com-jmespath-go-jmespath))
-    (arguments
-     '(#:import-path "github.com/aws/aws-sdk-go"
-       #:phases %standard-phases))
-    (synopsis "The official AWS SDK for the Go programming language")
-    (description
-     "The official AWS SDK for the Go programming language.")
-    (home-page "https://github.com/aws/aws-sdk-go")
-    (license license:asl2.0)))
-
 (define-public go-gopkg-in-ini
   (package
     (name "go-gopkg-in-ini")
-- 
2.41.0

